src/auth-service/bin/www

```diff
@@ -30,16 +51,74 @@ try {
 }
 
 
-
 /****
  * pass the app instance to the middleware configuration
  */
 try {
     middlewareConfig(app);
 } catch (e) {
-    console.log("middleware error", e.message);
+    logElement("middleware error", e.message)
 }
 
+
+const consumer = kafka.consumer({ groupId: constants.UNIQUE_CONSUMER_GROUP });
+
+// Function to start consuming messages
+const run = async() => {
+    await consumer.connect();
+    await consumer.subscribe({ topic: constants.NEW_MOBILE_APP_USER_TOPIC, fromBeginning: true });
```

Comment:
Hi @BenjaminSsempala , this is the topic we are using.

Reply:
Thanks @Baalmart